algo/src/main/java/com/hartwig/serve/sources/ckb/CkbReader.java

```diff
@@ -47,17 +48,18 @@ private static List<CkbEntry> curate(@NotNull List<CkbEntry> ckbEntries) {
     }
 
     @NotNull
-    private static List<CkbEntry> filter(@NotNull List<CkbEntry> entries, @NotNull List<CkbFilterEntry> ckbFilterEntries) {
-        CkbFilter filter = new CkbFilter(ckbFilterEntries);
+    private static List<CkbEntry> blacklist(@NotNull List<CkbEntry> entries,
+            @NotNull List<CkbBlacklistMolecularProfileEntry> ckbBlacklistEntries) {
+        CkbMolecularProfileBlacklistModel blacklist = new CkbMolecularProfileBlacklistModel(ckbBlacklistEntries);
 
-        LOGGER.info("Filtering {} CKB entries", entries.size());
-        List<CkbEntry> filteredEntries = filter.run(entries);
-        LOGGER.info(" Finished CKB filtering. {} entries remaining, {} entries have been removed",
-                filteredEntries.size(),
-                entries.size() - filteredEntries.size());
+        LOGGER.info("Blacklisting {} CKB entries", entries.size());
+        List<CkbEntry> blacklistedEntries = blacklist.run(entries);
```

Comment:
This function should return the whitelisted entries rather than blacklisted entries!

Reply:
good point, have renamed it to whitelisted entries